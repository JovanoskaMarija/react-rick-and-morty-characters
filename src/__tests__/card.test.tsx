import React from 'react'
import CountryCard from '../components/Card/Card'

import { render } from '@testing-library/react'

const character = {
  id: 1,
  name: 'Rick Sanchez',
  status: 'Alive',
  species: 'Human',
  type: '',
  gender: 'Male',
  image: 'https://rickandmortyapi.com/api/character/avatar/1.jpeg',
  episodes: 23,
}

describe('card', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<CountryCard character={character} />)

    expect(asFragment()).toMatchSnapshot()
  })
})
