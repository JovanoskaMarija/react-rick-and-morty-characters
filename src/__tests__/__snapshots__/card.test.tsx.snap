// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <img
      alt="Rick Sanchez"
      src="https://rickandmortyapi.com/api/character/avatar/1.jpeg"
    />
    <div
      class="info-card"
    >
      <span
        class="character-name"
      >
        Rick Sanchez
      </span>
      <div>
        <div
          class="info"
        >
          <span
            class="label"
          >
            gender:
          </span>
          <span
            class="value"
          >
            Male
          </span>
        </div>
        <div
          class="info"
        >
          <span
            class="label"
          >
            species:
          </span>
          <span
            class="value"
          >
            Human
          </span>
        </div>
        <div
          class="info"
        >
          <span
            class="label"
          >
            status:
          </span>
          <span
            class="value"
          >
            Alive
          </span>
        </div>
        <div
          class="info"
        >
          <span
            class="label"
          >
            episodes:
          </span>
          <span
            class="value"
          >
            23
          </span>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
